contracts/GenArt721CoreV3.sol

```diff
@@ -223,11 +223,26 @@ contract GenArt721CoreV3 is
     /// default base URI to initialize all new project projectBaseURI values to
     string public defaultBaseURI;
 
+    modifier onlyNonZeroAddress(address _address) {
+        require(_address != address(0), "Must input non-zero address");
+        _;
+    }
```

Comment:
nit: can we add test coverage for this?

Reply:
Good call-out, adding comprehensive test coverage for this.